qa.md

```diff
@@ -33,7 +33,7 @@
 ```
 
 
-## TSM03: Send an email to multiple To, CC and BCC
+## TSM03: Send an email to multiple To, CC and BCC.
```

Comment:
@ateska What's the reason behind these numbering. Why there are multiple TSM03's.?

Reply:
Discussed yesterday.